plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -119,6 +119,15 @@ const MemberComponent = (props: { member: UserEntity }) => {
   );
 };
 
+const useListStyles = makeStyles(() => ({
+  root: {
+    height: '100%',
```

Comment:
It's a bit odd that this thing renders a Grid item itself (not your fault of course). It makes the effects of this change a bit more complex to understand.

It would make sense to me if there was a maxHeight on something here, clamping large groups to show at most N rows and then the rest scrolls.

Reply:
MaxHeight can be set on parent Grid item is this make sense?